lib/src/screens/session/widgets/arrow_back_screen.dart

```diff
@@ -96,8 +96,8 @@ class _ArrowBackState extends State<ArrowBack> with WidgetsBindingObserver {
 
   @override
   Future<void> didChangeAppLifecycleState(AppLifecycleState state) async {
-    // If the app is resumed remove the route with this screen from the stack.
-    if (state == AppLifecycleState.resumed) {
+    // Pop back to home screen when leaving the app (for example going back to the browser)
+    if (state == AppLifecycleState.paused) {
```

Comment:
The disadvantage of solving it this way, is that now the `HomeScreen` becomes visible for a short time on Android after a session is finished.

Reply:
When do you see this? I believe this is not caused by the code above since its part of the ArrowBackScreen which is not rendered on Android.

Reply:
Ah of course. I think it's related to debug mode then, because in the actual app it reacts much faster.